plaidml/bridge/openvino/ops/deformable_convolution.cpp

```diff
@@ -0,0 +1,611 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using namespace edsl;
+
+namespace PlaidMLPlugin {
+
+size_t compute_padding_before(size_t input_size, size_t off_size, size_t filter_size, size_t stride,
+                              plaidml::op::AutoPadMode autopad_mode, size_t pad_before, size_t pad_end,
+                              size_t dilation) {
+  size_t f = (dilation * (filter_size - 1)) + 1;
+  if (autopad_mode == plaidml::op::AutoPadMode::EXPLICIT) {
+    return pad_before;
+  }
+  if (autopad_mode == plaidml::op::AutoPadMode::VALID) {
+    return 0;
+  }
+  if (autopad_mode == plaidml::op::AutoPadMode::SAME_LOWER || autopad_mode == plaidml::op::AutoPadMode::SAME_UPPER) {
```

Comment:
Not sure if you change these to if..else will be better

Reply:
This is same as the function of compute_padding_and_input_size() in plaidml/edsl/op/lib/ops.cc.

Reply:
Could you please use switch case here?

Reply:
> This is same as the function of compute_padding_and_input_size() in plaidml/edsl/op/lib/ops.cc.
I think we need to find a better way to call functions from plaidml edsl oplib..

Reply:
the return type of the function compute_padding_and_input_size() is pair<TensorDim, TensorDim>, if i call the function, maybe i need to convert type to size_t.

Reply:
Please add comment before this function where it is based on (plaidml/op/lib/ops.cc.) and what is the changes you made.